### PR TITLE
#163408888 generates an access token

### PR DESCRIPTION
**Pull request by jackiegitari1234**
**What does this PR do?**
- Add a token for successful user login

**Description of Task to be completed**
- Ensure login endpoint generates an access token

**How should this be manually tested?**
- clone the repository to your local machine
- run the commands
-  `virtualenv venv` -  to create a virtual environment
 -   `source venv/bin/activate`  to activate a virtual environment
- `pip install -r requirements.txt` to install all the application requirements
 - `flask run`
-  use postman to test the endpoint

What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2235314
#163408888

*Screenshot*

tokn

### CODE REVIEW

**Comment by Felix45** [comment]
This looks good, you only need to add the status code and data of the logged in user in the response object.

**Comment by jackiegitari1234** [comment]
> This looks good, you only need to add the status code and data of the logged in user in the response object.

Thanks, am fixing that

File: app/api/v2/views/auth_view.py [hunk 1]
```diff
@@ -15,6 +19,8 @@
 
 
 SECRET_KEY = os.getenv("SECRET")
+# app.config['JWT_SECRET_KEY'] = SECRET_KEY  # Change this!
+# jwt = JWTManager(app)
```
Comment (RuthNjeri) [inline app/api/v2/views/auth_view.py:24]: Do not push commented code. If you are not using it then delete it.

File: app/api/v2/views/auth_view.py [hunk 2]
```diff
@@ -121,4 +127,6 @@ def login():
         abort(make_response(
             jsonify({"message": "Wrong Password"}), 400))
 
-    abort(make_response(jsonify({"message": "user logged in"}), 200))
+    # abort(make_response(jsonify({"message": "user logged in"}), 200))
```
Comment (RuthNjeri) [inline app/api/v2/views/auth_view.py:130]: Delete commented code

File: run.py [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
+    get_jwt_identity
+)
 
 # local imports
 from app import create_app
 
 config_name = os.getenv('APP_SETTINGS')
 app = create_app(config_name)
+SECRET_KEY = os.getenv("SECRET")
+app.config['JWT_SECRET_KEY'] = SECRET_KEY  # Change this!
```
Comment (RuthNjeri) [inline run.py:15]: You can set the JWT secret key in the environment.

**Pull request merged**